monkey/infection_monkey/monkey.py

```diff
@@ -289,16 +286,15 @@ def _setup_plugin_event_forwarder(self) -> PluginEventForwarder:
     def _build_master(self, relay_port: int):
         servers = self._build_server_list(relay_port)
         local_network_interfaces = get_network_interfaces()
+        on_island = self._running_on_island(local_network_interfaces)
+        logger.debug(f"This agent is running on the island: {on_island}")
```

Comment:
This doesn't have anything to do with building the master, so it shouldn't be in `_build_master()`. 

I don't think we really need this at all anymore. If we still want to log it, then right after `logger.info("Agent is starting...") is probably the best place.